### DISCUSSION TITLE
Solid js support

### DISCUSSION THREAD

**Discussion by tanekim88** [vercel/next.js#31076] [Ideas]
### Describe the feature you'd like to request

Solid js is smaller and more efficient than React while having simlar syntax with React. It would be awesome if Solid Js is supported as well.

### Describe the solution you'd like

Support Solid Js

### Describe alternatives you've considered

Support Solid Js

**Comment by sbfkcel** [comment]
Cool, react has too many repeated render calls. Solid JS is much more elegant.

**Comment by elijah629** [comment]
Wish this would happen, but I don't think it will. :disappointed:

**Comment by clicktodev** [comment]
Tanstack start supports react and solid
